views/data_summary.erb

```diff
@@ -0,0 +1,355 @@
+                    <br>
+                    &nbsp;
+                    <div>
+                    <table class="definition">
+                      <tr>
+                        <td><strong>CSV field</strong></td>
+                        <td><strong>use &amp; example value</strong></td>
+                      </tr>
+                      <tr>
+                        <th>id</th>
+                        <td>
+                          unique within EveryPolitician
+                          <em>for this politician</em> (so you can use it to track
+                          people across terms, for example)
+                          <br>
+                          It's a <a href="https://en.wikipedia.org/wiki/Universally_unique_identifier">UUID</a>,
+                          and looks like
+                          <br><code>2bc9cc09-a33a-42d9-89c3-14effb20b8b0</code>
+                        </td>
+                      </tr>
+                      <tr>
+                        <th>name</th>
+                        <td>
+                          common name for this politician: <code>Anne Example</code>
+                          <br>See <a href="#about-names">more about names</a>
+                        </td>
+                      </tr>
+                      <tr>
+                        <th>sort_name</th>
+                        <td>name for sort ordering: <code>Example, Anne</code></td>
+                      </tr>
+                      <tr>
+                        <th>email</th>
+                        <td>email address: <code>anne@example.com</code></td>
+                      </tr>
+                      <tr>
+                        <th>twitter</th>
+                        <td>twitter handle: <code>example</code> (for <code>@example</code>)</td>
+                      </tr>
+                      <tr>
+                        <th>facebook</th>
+                        <td>Facebook name: <code>AnneExample</code> (for <code>facebook.com/AnneExample)</code>
+                      </tr>
+                      <tr>
+                        <th>group</th>
+                        <td>name of political group <code>Example Faction</code></td>
+                      </tr>
+                      <tr>
+                        <th>group_id<span class="superscript">&dagger;</span></th>
+                        <td>id for group</td>
+                      </tr>
+                      <tr>
+                        <th>area</th>
+                        <td>name of area represented <code>Example County</code></td>
+                      </tr>
+                      <tr>
+                        <th>area_id<span class="superscript">&dagger;</span></th>
+                        <td>id for area</td>
+                      </tr>
+                      <tr>
+                        <th>chamber<span class="superscript">&dagger;</span></th>
+                        <td>name of chamber <code>House of Representatives</code></td>
```

Comment:
I'm not sure I'd include this. As the file only exists at the level of a single legislature, the value will be the same in every row here. At some point (soon?) we'll completely remove this. If this were the formal documentation for the file formats, we should probably include it and note that it's deprecated, but this doesn't seem like the right place for that…